pages/en/developing/supported-chains.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Supported Chains
+---
+
+## The Graph's Decentralized Network
+
+The following chains are supported on The Graph's Decentralized Network:
+
+- `mainnet` (Ethereum)
+- `gnosis`
+
+## Hosted Service
+
+The following chains are supported in beta on the Hosted Service:
+
+- `goerli`
+- `poa-core`
+- `poa-sokol`
+- `matic` (now known as Polygon)
+- `mumbai`
+- `fantom`
+- `bsc` (now known as BNB Chain)
+- `chapel`
+- `clover`
+- `avalanche`
+- `fuji`
+- `celo`
+- `celo-alfajores`
+- `fuse`
+- `moonriver`
+- `mbase`
+- `arbitrum-one`
+- `arbitrum-rinkeby`
+- `arbitrum-goerli`
+- `optimism`
+- `optimism-goerli`
+- `aurora`
+- `aurora-testnet`
+- `boba`
+- `harmony`
+- `zkSync2-testnet`
+
+You will **not be able** to publish a subgraph that indexes a non-mainnet network to the decentralized Graph Network in [Subgraph Studio](/deploying/subgraph-studio).
+
+## Graph Node
+
+If your preferred chain isn't support on The Graph's decentralized network, you can run your own Graph Node to index any EVM-compatible chain. Make sure that the [version](https://github.com/graphprotocol/graph-node/releases) you are using supports the network and you have the needed configuration.
+
+Graph Node can also index other protocols, via a Firehose integration. Firehose integrations have been created for NEAR, Arweave and Cosmos-based chains.
+
+> The hosted service relies on the stability and reliability of the underlying technologies, namely the provided JSON RPC endpoints. Newer networks will be marked as being in beta until the network has proven itself in terms of stability, reliability, and scalability. During this beta period, there is a risk of downtime and unexpected behaviour.
```

Comment:
> The hosted service relies on the stability and reliability of the underlying technologies, namely the provided JSON RPC endpoints. 
This refers to the hosted service

> Newer networks will be marked as being in beta until the network has proven itself in terms of stability, reliability, and scalability. During this beta period, there is a risk of downtime and unexpected behaviour.
This refers to the decentralized network.

This might be unclear to the reader. Perhaps we should split the disclaimers - and add them to their respective sections?

Reply:
+ Make sure these network are all in the list https://github.com/graphprotocol/docs/issues/269

Reply:
Trimmed & moved the disclaimer message.

Also confirmed all those networks are listed on this page.